services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -102,14 +110,15 @@ export const ChoiceListField = ({
               if (child.props?.choices) {
                 child.props.choices.forEach((choice: FieldChoice) => {
                   choice.checked = false;
-                  form.setValue(child.id, []);
                 });
+                form.setValue(child.id, []);
```

Comment:
Little did we know that if you call this multiple times validation gets very upset and gets wild. Thats why its moved out of the forEach loop.